data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -131,6 +131,59 @@
 # 60: American Samoa, 66: Guam, 69: N. Mariana Islands, 78: US Virgin Islands
 TILES_ISLAND_AREA_FIPS_CODES = ["60", "66", "69", "78"]
 TILES_PUERTO_RICO_FIPS_CODE = ["72"]
+TILES_NATION_FIPS_CODE = [
```

Comment:
I feel unusually happy these are here now

Reply:
a pretty dumb question -- why not just do a list comprehension dropping 52?